draft-ietf-quic-recovery.md

```diff
@@ -320,10 +320,17 @@ SHOULD be set for twice the newly computed smoothed RTT.
 
 When crypto packets are sent, the sender SHOULD set a timer for the crypto
 timeout period.  Upon timeout, the sender MUST retransmit all unacknowledged
-CRYPTO data by calling RetransmitAllUnackedCryptoData(). On each consecutive
-expiration of the crypto timer without receiving an acknowledgement for a new
-packet, the sender SHOULD double the crypto handshake timeout and set a timer
-for this period.
+CRYPTO data if possible.
+
+Until the server has confirmed the path, it must not send more than 3
```

Comment:
rather than "confirmed the path", we should be more consistent and say "the server has validated the client's address on the path"

```suggestion
Until the server has validated the client's address on the path, it must not send more than 3
```